### PR TITLE
feat/backend/database/register

### PR DESCRIPTION
**Pull request by bbcarrots**
- checked that db insert function works
- cleaned up code
- put comments

### CODE REVIEW

**Review by allainerain — COMMENTED** [review]
Just a question, is the connectAdminMySQL tested already?

Also, I feel like in the future, we should keep the user and the password variables as private variables in a .env file for security purposes.

Waiting for @Weejee01's  approval for this PR.

**Comment by bbcarrots** [comment]
I resolved some conflicts in the config files.

Regarding Allaine's questions:
- Admin has not yet been tested.
- I can set up the .env files as a fix in the next branch!

**Review by Weejee01 — APPROVED** [review]
Function for inserting works!

**Pull request merged**
